Makefile

```diff
@@ -62,6 +63,13 @@ artifact: build pack docs
 
 lint: $(NPM_INSTALL_STAMP) build
 	$(ESLINT) .
+	$(TSC) --noEmit --project tsconfig.json
+	$(PRETTIER) --check .
```

Comment:
Any reason you don't want to just use prettier via eslint directly? My experience has been that it works well (including via  `--fix`).

Reply:
I was going to do that, but the Prettier docs recommend [not doing that](https://prettier.io/docs/en/integrating-with-linters.html#notes):

> When searching for both Prettier and your linter on the Internet you’ll probably find more related projects. These are **generally not recommended** [...]